net-misc/uftp/uftp-4.10.1.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="Encrypted UDP based FTP with multicast"
+HOMEPAGE="http://uftp-multicast.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}-multicast/${P}.tar.gz"
```

Comment:
You should switch to using *http://downloads.sourceforge.net* instead, since its a mirroring service of their own.